src/afl-forkserver.c

```diff
@@ -40,12 +40,18 @@
 #include <sys/wait.h>
 #include <sys/resource.h>
 
+#if defined(__APPLE__) || defined(__FreeBSD__) || defined(__OpenBSD__)
+#define HAVE_ARC4RANDOM 1
+#endif                           /* __APPLE__ || __FreeBSD__ || __OpenBSD__ */
+
```

Comment:
Should HAVE_ARC4RANDOM maybe be moved to `include/config.h` instead?

Reply:
in my repository state it is in ./include/afl-fuzz.h which is fine

Reply:
ah sorry, now I get it. yes I will move it.

Reply:
will move it to config.h